assets/aircraft/hdjt.json

```diff
@@ -5,16 +5,16 @@
         "number": 2,
         "type": "J"
     },
-    "weightClass": "S",
+    "weightClass": "L",
```

Comment:
Honda Jet is a small. Reverting this.

Reply:
done

Reply:
![screenshot_211](https://user-images.githubusercontent.com/24833249/52430841-45f59a80-2b07-11e9-8080-690f441efec0.png)
never heard of the weightclass small so I looked it up and it said light that why I changed it 🤔

Reply:
Bizarre, I would have thought ICAO would at least go with M... the FAA has it as a small:

![image](https://user-images.githubusercontent.com/5103735/52431421-33af3880-2ad6-11e9-9da8-02697ea88797.png)

It weighs 7,200 pounds, and small is <12,500. So the small is correct I guess, though it's real strange that your screenshot shows it as a large. 🤷‍♂️